### ISSUE TITLE
rotating image causes app to crash

### ISSUE THREAD

**Issue by not-Ambrose** [files-community/Files#8658]
### Description

The apps crashing when you try to rotate the image

### Steps To Reproduce

1. Find a image
2. After you find it, click on it (but don't open it)
3. Fullscreen files 
4. Double click the "rotate image" button
5. Congrats! The app has crashed

### Expected behavior

I expected the image to rotate twice

### Files Version

Version: 2.1.15.0 OS Version: 10.0.22000.556

### Windows Version

Windows 11 Home 10.0.22000 Build 22000

### Relevant Assets/Logs

[debug_fulltrust.log](https://github.com/files-community/Files/files/8242993/debug_fulltrust.log)
[debug.log](https://github.com/files-community/Files/files/8242996/debug.log)

**Comment by devovercome** [comment]
Hey everyone! May I fix that? 

TL; DR;
I think, the [AsyncRelayCommand](https://github.com/files-community/Files/blob/bf8c9e2e80b625cd859415a2e10211bed8521b46/src/Files.Uwp/Interacts/BaseLayoutCommandsViewModel.cs#L76) should be debounced somehow, by using the [IsRunning](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.mvvm.input.asyncrelaycommand.isrunning?view=win-comm-toolkit-dotnet-7.0#microsoft-toolkit-mvvm-input-asyncrelaycommand-isrunning) property. I gonna look for WPF native ways to perform that (there should be a way to disable a button while command `IAsyncRelayCommand.IsRunning == true` or bind it to `ICommand.CanExecute`)

**Comment by yaichenbaum** [comment]
> Hey everyone! May I fix that?
> 
> TL; DR; I think, the [AsyncRelayCommand](https://github.com/files-community/Files/blob/bf8c9e2e80b625cd859415a2e10211bed8521b46/src/Files.Uwp/Interacts/BaseLayoutCommandsViewModel.cs#L76) should be debounced somehow, by using the [IsRunning](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.mvvm.input.asyncrelaycommand.isrunning?view=win-comm-toolkit-dotnet-7.0#microsoft-toolkit-mvvm-input-asyncrelaycommand-isrunning) property. I gonna look for WPF native ways to perform that (there should be a way to disable a button while command `IAsyncRelayCommand.IsRunning == true` or bind it to `ICommand.CanExecute`)

@devovercome sure 👍

**Comment by yaichenbaum** [comment]
This issue should be resolved with #8682
